src/site/content/en/blog/bfcache/index.md

```diff
@@ -230,12 +230,14 @@ time](https://developer.chrome.com/blog/page-lifecycle-api/#the-unload-event)).
 So browsers are faced with a dilemma, they have to choose between something that
 can improve the user experience—but might also risk breaking the page.
 
-Chrome and Firefox have chosen to make pages ineligible for bfcache if they add an `unload`
+On desktop, Chrome and Firefox have chosen to make pages ineligible for bfcache if they add an `unload`
 listener, which is less risky but also disqualifies _a lot_ of pages. Safari
 will attempt to cache some pages with an `unload` event listener, but to reduce
 potential breakage it will not run the `unload` event when a user is navigating
 away, which makes the event very unreliable.
 
+On mobile, most browsers will attempt to cache pages with an `unload` event listener since the risk of breakage is lower due to the fact that the `unload` event has always been extremely unreliable on mobile.
```

Comment:
Can we be more specific that Chrome is one of these browsers? Part of the motivation for #9044 was to be able to cite this article as a source for explaining the CLS changes in CrUX, so explicitly connecting it back to Chrome would be helpful.

Reply:
The previous paragraph specifically calls out Chrome and Firefox for making pages ineligible, so it seems ok to clarify known exceptions where that's not the case.

Agree MDN should be the canonical source for exact browser behavior.

Reply:
Works for me.

Reply:
Cool I'll tee up a new PR and add you both as reviewers